README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/lalithsuresh/hiero.svg?branch=master)](https://travis-ci.org/lalithsuresh/hiero)
```

Comment:
You are a commiter, so why is that necessary?
Should we wait until we get a repo in the vmware org?
Also, the UI part has no unit tests :( But we can try to at least build it.